keyboards/hub16/rules.mk

```diff
@@ -0,0 +1,78 @@
+#F_USB after it has run through any CPU prescalers.Note that this value
+#does not*change * the processor frequency - it should merely be updated to
+#reflect the processor speed set externally so that the code can use     accurate
+#software delays.
+F_CPU = 16000000
+
+#LUFA specific
+#
+#Target architecture(see library "Board Types" documentation).
+ARCH = AVR8
+
+#Input clock        frequency.
+#This will define a symbol, F_USB, in all source code files equal to the
+#input clock frequency(before any prescaling is performed) in Hz.This value may
+#differ from F_CPU if prescaling is used on the latter, and is required as the
+#raw input clock is fed directly to the PLL sections of the AVR for high speed
+#clock generation for the USB and other AVR subsections.Do NOT tack on a 'UL'
+#at the end, this will be done automatically to create a 32 - bit value in your
+#source code.
+#
+#If no clock division is performed on the input clock inside the AVR(via the
+#CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+#Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+#Bootloader selection
+#Teensy halfkay
+#Pro Micro caterina
+#Atmel DFU atmel - dfu
+#LUFA DFU  lufa - dfu
+#QMK DFU   qmk - dfu
+#atmega32a bootloadHID
+BOOTLOADER = caterina
+
+#If you            don't know the bootloader type, then you can specify the
+#Boot Section Size in* bytes* by uncommenting out the OPT_DEFS line
+#Teensy halfKay 512
+#Teensy++ halfKay 1024
+#Atmel DFU loader 4096
+#LUFA bootloader 4096
+#USBaspLoader 2048
+#OPT_DEFS += -DBOOTLOADER_SIZE = 4096
+
+#Build Options
+#change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = no       # Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE = yes        # Console for debug(+400)
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+#Do not enable SLEEP_LED_ENABLE.it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+#if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
+ENCODER_ENABLE = yes		# Rotary Encoder support
+TAP_DANCE_ENABLE = yes 		# Support for tap dancing
```

Comment:
Please replace Lines 57-78 with the following:

```Makefile
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
UNICODE_ENABLE = no         # Unicode
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
ENCODER_ENABLE = yes        # Rotary Encoder support
TAP_DANCE_ENABLE = yes      # Support for tap dancing
```

This removes the firmware size impact notes, which we're trying to get rid of, and aligns the comments.